easy_phi/templates/base.html

```diff
@@ -3,12 +3,12 @@
 <head>
     <meta content="text/html;charset=utf-8" http-equiv="Content-Type">
     <title>{% block title %}Easy Phi - test measurement equipment platform{% end %}</title>
-    {% block css %}
```

Comment:
we will need some way to add page-specific css later. It is not used at the moment